CorsixTH/Lua/world.lua

```diff
@@ -954,6 +954,12 @@ function World:pauseOrUnpause()
   end
 end
 
+--! Function to check if player can perform actions when paused
```

Comment:
Description and function name are slightly off. Maybe simply `isUserActionProhibited()` would be more appropriate. Pause and the config value are not necessarily important details to the caller.

Reply:
This has been updated